### ISSUE TITLE
[Feature Request] Allow Custom Name Entry in 2SP mode

### ISSUE THREAD

**Issue by Sonic-TV** [emoose/OutRun2006Tweaks#46]
Name entry screen should be in 2SP mode but it's not.
The system will simply take first 4 letters.
Example: My username will be taken as "Soni" in 2SP "Best Outrunners" Screen.

**Comment by lance8765** [comment]
Yeah i've always hated that, my real name has more than 4 letters, too.

It takes the name from the Driver License though...

**Comment by Sonic-TV** [comment]
You can't use the Rankings screen for IR as well. The IR page requires you to have the IR screen for score registration.

**Comment by Sonic-TV** [comment]
in case I haven't mentioned yet, there're leftover graphics for the name entry screen in the C2C Sprite folder.
